src/sentry/api/event_search.py

```diff
@@ -760,35 +758,59 @@ def visit_text_value(self, node, children):
     def visit_search_value(self, node, children):
         return SearchValue(children[0])
 
-    # def visit_numeric_value
-    # def visit_boolean_value
-    # def visit_text_in_list
-    # def visit_numerc_in_list
+    def visit_numeric_value(self, node, children):
+        return node
+
+    def visit_boolean_value(self, node, children):
+        return node
+
+    def visit_text_in_list(self, node, children):
+        return process_list(children[1], children[2])
+
+    def visit_numeric_in_list(self, node, children):
+        return process_list(children[1], children[2])
 
     def visit_iso_8601_date_format(self, node, children):
         return node.text
 
-    # def visit_rel_date_format
-    # def visit_duration_format
-    # def visit_percentage_format
+    def visit_rel_date_format(self, node, children):
+        return node
+
+    def visit_duration_format(self, node, children):
+        return node
+
+    def visit_percentage_format(self, node, children):
+        return node
 
     def visit_operator(self, node, children):
         return node.text
 
-    # def visit_or_operator
-    # def visit_and_operator
+    def visit_or_operator(self, node, children):
+        return node.text.upper()
+
+    def visit_and_operator(self, node, children):
+        return node.text.upper()
 
     def visit_open_paren(self, node, children):
         return node.text
 
     def visit_closed_paren(self, node, children):
         return node.text
 
-    # def visit_open_bracket
-    # def visit_closed_bracket
-    # def visit_sep
-    # def visit_negation
-    # def visit_comma
+    def visit_open_bracket(self, node, children):
+        return node.text
+
+    def visit_closed_bracket(self, node, children):
+        return node.text
+
+    def visit_sep(self, node, children):
+        return node
+
+    def visit_negation(self, node, children):
+        return node
+
+    def visit_comma(self, node, children):
+        return node
```

Comment:
Why not return `node.text` for all of these? Otherwise won't we still be hitting `generic_visit` again?

Reply:
it won't hit the generic visit, as long as we have visitors for all the types of nodes we don't need the generic visitor (though we still need it right now for optional nodes)

Reply:
Also we can't translate some of these to text without making some other updates, which I didn't _really_ want to do